File: sandbox/python/src/TeamCreateExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    team_create_request = models.TeamCreateRequest(
        name="New Team Name",
    )

    try:
        response = api.TeamApi(api_client).team_create(
            team_create_request=team_create_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling TeamApi#team_create: %s\n" % e)

File: sandbox/php/src/TeamCreateExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
$config->setUsername("YOUR_API_KEY");
// $config->setAccessToken("YOUR_ACCESS_TOKEN");

$team_create_request = (new Dropbox\Sign\Model\TeamCreateRequest())
    ->setName("New Team Name");

try {
    $response = (new Dropbox\Sign\Api\TeamApi(config: $config))->teamCreate(
        team_create_request: $team_create_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling TeamApi#teamCreate: {$e->getMessage()}";
}
